subsys/caf/modules/sensor_manager.c

```diff
@@ -326,15 +325,19 @@ static int sensor_trigger_init(const struct sm_sensor_config *sc, struct sensor_
 
 	reset_sensor_sleep_cnt(sc, sd);
 
-	LOG_INF("Trigger configured (threshold: %d, time: %d ms)",
-		(int)(1000 * sc->trigger->activation.thresh),
+	LOG_INF("Trigger configured (threshold: %d,%d time: %d ms)",
+		sc->trigger->activation.thresh.val1,
+		sc->trigger->activation.thresh.val2,
 		sc->trigger->activation.timeout_ms);
 	return 0;
 }
 
+#endif /* CONFIG_CAF_SENSOR_MANAGER_PM */
+
 static void configure_max_power_state(void)
 {
-	if (IS_ENABLED(CONFIG_CAF_SENSOR_MANAGER_ACTIVE_PM)) {
```

Comment:
What is the reason of this all changes from C conditionals to preprocessor conditionals?